### ISSUE TITLE
Manage drop-in files

### ISSUE THREAD

**Issue by jarro2783** [camptocamp/puppet-systemd#15]
Can support for drop-in files to override package units be added to this module?

**Comment by raphink** [comment]
Could you give a concrete example of what you need?

**Comment by jarro2783** [comment]
If I want to override some parameters for a systemd unit, then I would put a drop in file in `/etc/systemd/system/unit.service.d`, with the parameters that I want to override. So perhaps I could do something like:
```
systemd::unit_dropin { '99environment':
  unit => 'unitname',
  content => "[Service]\nStandardOutput=syslog\n",
}
```
and the contents would get placed in `/etc/systemd/system/unitname/99environment.conf`.

**Comment by gauthier-delacroix** [comment]
That would be a nice addition to customize systemd services without overriding the entire original file.

**Comment by TheMeier** [comment]
can this be closed now? and is PR #26 still needed at all?

**Comment by jarro2783** [comment]
Yeah I think this is resolved by #39.
